### ISSUE TITLE
HNC: publish basic metrics

### ISSUE THREAD

**Issue by adrianludwin** [kubernetes-sigs/multi-tenancy#111]
See https://kubebuilder.io/reference/metrics.html. We need to also write a mini-design doc (possibly write in this issue?) stating which metrics we need to collect.

/assign @lafh

**Comment by lafh** [comment]
update: we decide to move forward with [OpenTelemetry](https://opentelemetry.io/), the merged version of OpenCensus and OpenTracing. The goal of OpenTelemetry is to standardize metric collection and distribution, and it is now the defacto standard for observability instrumentation.

**Comment by rjbez17** [comment]
OpenTelemtry is pretty broad and requires you to implement the exporters within your code. Have we decided on the initial exports we plan to support? I suggest Prometheus as it is the only natively supported metrics implementation for core kubernetes (that I'm aware of). The currently supported metrics exporters are here: 

https://github.com/open-telemetry/opentelemetry-go/tree/master/exporter/metric

TL;DR: A few statsd variants, prometheus, and stdout.

**Comment by adrianludwin** [comment]
/assign @yiqigao217 

Fixed by #396, #410 and #418.

/close
